keps/sig-auth/20190607-certificates-api.md

```diff
@@ -132,6 +133,31 @@ A typical successful issuance proceeds as follows.
    the `.Status.Certificate` field.
 1. The requestor observes the update, and stores the certificate locally.
 
+### Signers
+CSRs have a `signerName` field which is used to specify which signer the CSR creator wants to sign the certificate.
+To support migration from v1beta1 to v1, this required field will be defaulted in v1beta1, but not in v1:
```

Comment:
Clarify the field would be optional (`+optional`, not required in openapi) in v1beta1.

Reply:
Could reiterate the `spec.signerName` field is immutable (I guess it's covered by the `The information contained in the spec is immutable after the request is created.` statement above)